examples/blockchain/network.rs

```diff
@@ -372,8 +372,14 @@ impl NetworkManager {
                     }
                 }
                 event = self.swarm.next().fuse() => {
```

Comment:
There is also the second example: `dummy_honest.rs` -- if we want to maintain it, then this likely also needs to be changed there.

Another option would be to drop the `dummy_honest` example :)

Reply:
I think I support dropping it – if we are planning to simplify and improve (i.e. remove libp2p ;P) the main example then it should be enough for any documentation/tutorial purposes.

Reply:
Fixed :)

Reply:
I think that is the most disgusting solution you could have picked. xd

Not that I object much.